Documentation/k8s-monitoring.md

```diff
@@ -5,40 +5,51 @@ Note that these steps work best with a local Kubernetes cluster running in `Vagr
 ## Prometheus Operator
 First the Prometheus operator needs to be started in the cluster so it can watch for our requests to start monitoring Rook and respond by deploying the correct Prometheus pods and configuration.
 A full explanation can be found in the [Prometheus operator repository on github](https://github.com/coreos/prometheus-operator), but the quick instructions can be found here:
-```
-wget -qO- https://raw.githubusercontent.com/coreos/prometheus-operator/master/deployment.yaml | kubectl create -f -
+```bash
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml
```

Comment:
good question. We could snap to the latest prometheus release, but then we have to worry about updating it periodically. At least we wouldn't be surprised by breaking changes